sandbox-server/index.js

```diff
@@ -0,0 +1,19 @@
+const express = require('express')
```

Comment:
I like that ya'll didn't use express generator. I think this is a great way to see the underlying simplicity of creating a http server